metricbeat/.gitignore

```diff
@@ -7,3 +7,4 @@ _meta/beat.full.yml
 /metricbeat.test
 /docs/html_docs
 
+target/
```

Comment:
Yes. As long as there is no other directory named target we shouldn't have any issues.